src/associations/belongs-to.d.ts

```diff
@@ -1,8 +1,8 @@
 import { DataType } from '../data-types';
-import { CreateOptions, CreationAttributes, FindOptions, Model, ModelCtor, SaveOptions } from '../model';
+import { CreateOptions, CreationAttributes, FindOptions, Model, ModelStatic, SaveOptions } from '../model';
 import { Association, AssociationOptions, SingleAssociationAccessors } from './base';
 
-// type ModelCtor<M extends Model> = InstanceType<typeof M>;
+// type ModelStatic<M extends Model> = InstanceType<typeof M>;
```

Comment:
Do we need this?

Reply:
Nope, but it's being removed in https://github.com/sequelize/sequelize/pull/14280 :)